src/components/FormNav.js

```diff
@@ -16,23 +16,25 @@ const FormNav = ({ prevPath, nextPath, submit, handlePageChange }) => {
         {
           prevPath && prevPath.length
             ? <StyledButton
-                onClick={() => {
-                  handlePageChange(prevPath);
-                }
-              }>
+                onClick={
```

Comment:
I recommend defining `onClickPrev` and `onClickNext` above instead of inlining the function handlers here. I still need to do more research on this, but I * think * handlers as inline functions are less performant.

Reply:
Updated